src/main/webapp/WEB-INF/views/404Error.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>404 Not Found Page</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+		
+    <link href="http://netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/css/bootstrap-combined.min.css" rel="stylesheet">
+    <link href="css/error.css" rel="stylesheet">
+	
+    <script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+    <script src="http://netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/js/bootstrap.min.js"></script>
+</head>
+
+<body>
+<div class="container">
+    <div class="row">
+    <div class="error-template">
+	    <h1>Oops!</h1>
+	    <h2>404 Not Found</h2>
+	    <div class="error-details">
+		Sorry, an error has occured, Requested page not found!<br>
+	    </div>
+	    <div class="error-actions">
+		<a href="DesignPage.html" class="btn btn-primary">
+		    <i class="icon-home icon-white"></i> Take Me Home </a>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)